uber/model_checks.py

```diff
@@ -120,7 +120,7 @@ def emergency_contact(attendee):
 @validation.Attendee
 @ignore_unassigned_and_placeholders
 def cellphone(attendee):
-    if attendee.cellphone and _invalid_phone_number(attendee.cellphone):
+    if attendee.cellphone and _invalid_phone_number(attendee.cellphone) and not c.COLLECT_FULL_ADDRESS:
```

Comment:
See my comment on your other PR that (accidentally?) included these changes:
https://github.com/magfest/ubersystem/pull/1566/files#r42947566